docs/src/transaction.md

```diff
@@ -1,25 +1,29 @@
 # Transactions
 
-A `Transaction` in Miden is the state transition of a single account. A `Transaction` takes as input a single [account](account.md) and zero or more [notes](note.md), and outputs the same account with an updated state, together with zero or more notes. `Transaction`s in Miden are Miden VM programs, their execution resulting in the generation of a zero-knowledge proof.
+A `Transaction` in Miden is the state transition of a single account. A `Transaction` takes as input a single [account](account.md) and zero or more [notes](note.md), and outputs the same account with an updated state, together with zero or more notes. Transactions in Miden are Miden VM programs, their execution resulting in the generation of a zero-knowledge proof.
 
 Miden's `Transaction` model aims for the following:
 
 - **Parallel transaction execution**: Accounts can update their state independently from each other and in parallel.
-- **Private transaction execution**: Client-side `Transaction` proving allows the network to verify `Transaction`s validity with zero knowledge.
+- **Private transaction execution**: Client-side `Transaction` proving allows the network to verify transactions validity with zero knowledge.
```

Comment:
Same comment as above. Maybe emphasizing transaction always is too much?

Reply:
Yeah - I don't think we need to emphasize "transaction" everywhere (or even in most places). It actually detracts from readability a bit.